src/Commands/MigrateFresh.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Console\Migrations\MigrateCommand;
+use Illuminate\Database\Migrations\Migrator;
+use Stancl\Tenancy\DatabaseManager;
+use Stancl\Tenancy\Traits\DealsWithMigrations;
+use Stancl\Tenancy\Traits\HasATenantsOption;
+
+final class MigrateFresh extends MigrateCommand
+{
+    use HasATenantsOption, DealsWithMigrations;
+
+    protected $database;
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Drop all tables and re-run all migrations for tenant(s)';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(Migrator $migrator, DatabaseManager $database)
+    {
+        parent::__construct($migrator);
+        $this->database = $database;
+
+        $this->setName('tenants:migrate-fresh');
+        $this->specifyParameters();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        if (! $this->confirmToProceed()) {
+            return;
+        }
+
+        $originalTenant = tenancy()->getTenant();
+        tenancy()->all($this->option('tenants'))->each(function ($tenant) {
+            $this->line("Tenant: {$tenant['id']}");
+
+            // See Illuminate\Database\Migrations\DatabaseMigrationRepository::getConnection.
+            // Database connections are cached by Illuminate\Database\ConnectionResolver.
+            $this->input->setOption('database', 'tenant');
+            tenancy()->initialize($tenant);
+
+            // Fresh
+            $this->call('db:wipe', array_filter([
```

Comment:
Why the `array_filter`?

Reply:
That's a fragment from Laravel's `wipe` command. It should have been removed when I removed the other options that it filters against.

Reply:
I've added `array_filter` to `db:wipe` just because other Laravel commands has it in programmable executed calls:
```php
$this->call('migrate:install', array_filter([
    '--database' => $this->option('database'),
]));

$this->call('db:seed', array_filter([
    '--database' => $database,
    '--class' => $this->option('seeder') ?: 'DatabaseSeeder',
    '--force' => true,
]));

```

Reply:
I see. It's probably because it will remove the `--force` if it's false. I'll add that to the other PR.

Reply:
`array_filter` was introduced in this PR: https://github.com/laravel/framework/pull/27020

Reply:
Makes sense. Thanks for the link @antonkomarev.